### ISSUE TITLE
Import files from skipped folders

### ISSUE THREAD

**Issue by shshaw** [bdkjones/CodeKit#53]
**Quick, short summary:**
Telling CodeKit to ignore `/node_modules/` and other certain directories is quite sensible. However, say I have jQuery or GSAP installed via NPM that I want to compile into a `plugins.js` file using `// @codekit-append /node_modules/gsap/src/uncompressed/TweenMax.js`, etc. According to [issue 395 on CodeKit 2](https://github.com/bdkjones/CodeKit/issues/395), this should be possible, but in actuality does not work.

**Expected results:**
Files imported with `@codekit-append` and `@codekit-prepend` will be inserted when the file containing those import statements is processed, whether they are in a skipped folder or not.

To be clear, I am not expecting CodeKit to process `plugins.js` whenever a file in a skipped folder is changed. I am expecting CodeKit to include the files from skipped folders whenever `plugins.js` is processed, either manually with the Process button or by saving `plugins.js`.

**Actual results:**
Files in the skipped folder are not included into the processed file that contains the `@codekit-append` and `@codekit-prepend` statements. Files which are not in skipped folders are included.

**Exact steps to reproduce:**
1. Tell CodeKit to skip a folder
2. Use  `@codekit-append` or `@codekit-prepend` to try to import a file from the skipped folder
3. See that the processed file does not contain the file from the skipped folder.

**A link to download a simplified project or file that shows the issue:**
[Skipped File Test.zip](https://github.com/bdkjones/codekit3/files/278194/Skipped.File.Test.zip)

**Comment by bdkjones** [comment]
The oldest open issue on the tracker! There's a dupe in #419 that I have commented on, but:

(1) Errors will now be shown for files missing from the chain.

(2) Support for using items that are in Skipped Folders (or even outside of the project, as long as the app can map the import statement to a file on disk) now work. 

Closing this one and tracking in #419. Sorry this took such a long time to implement; it was major surgery and there was a time I was debating whether it was even worth it since ECMA2015 has module syntax. In the end, I decided I can't drop the `prepend/append` workflows without a long deprecation process.
